packages/scandipwa/src/component/CheckoutDeliveryOption/CheckoutDeliveryOption.component.js

```diff
@@ -51,14 +53,12 @@ export class CheckoutDeliveryOption extends PureComponent {
                 price_excl_tax
             },
             totals: {
-                cart_display_config: {
-                    display_tax_in_shipping_amount
-                } = {},
                 quote_currency_code
-            }
+            },
+            displayTaxInPrice
         } = this.props;
 
-        if (display_tax_in_shipping_amount === DISPLAY_CART_TAX_IN_SHIPPING_EXL_TAX) {
+        if (displayTaxInPrice === DISPLAY_SHIPPING_PRICES_EXCL_TAX) {
```

Comment:
Display Shipping Prices in Price Display Settings defines if shipping price will include/exclude tax or display both.